source/components/ThemePageControl.cpp

```diff
@@ -165,7 +165,7 @@ __fastcall TThemePageControl::TThemePageControl(TComponent * Owner) :
 //----------------------------------------------------------------------------------------------------------
 int __fastcall TThemePageControl::GetTabsHeight()
 {
-  // Calculated height includes tab/contents separator line on Windows 7/8,

```

Comment:
This was correct, or rather it should be "The calculated height includes..."

Reply:
Made the changes for this can you please review it now